tests/kernel/timer/timer_api/Kconfig

```diff
@@ -4,4 +4,7 @@
 config SYS_CLOCK_TICKS_PER_SEC
 	default 16384 if NRF_RTC_TIMER && TICKLESS_KERNEL
 
+config ASSERT_VERBOSE
```

Comment:
As Kconfig variables are in the global namespace, this name is too generic IMO. You should prefix it with something that limits it to the timer API namespace.